### ISSUE TITLE
Url Whitelist doesn't work

### ISSUE THREAD

**Issue by Aushilfsarnie** [adhell/adprovider#4]
Hello, 
thank you for great App, but url-whitelist doesn't work.

**Comment by raiym** [comment]
Please provide example

**Comment by Aushilfsarnie** [comment]
http://cdn1.smartadserver.com/

**Comment by Aushilfsarnie** [comment]
Everything in ad-url-ausschlussliste is blocked

**Comment by raiym** [comment]
Have you tried to disable and enable it again?

**Comment by Aushilfsarnie** [comment]
Yes, also reboot and installed it again, but Everything is blocked

**Comment by Aushilfsarnie** [comment]
Thanks for your answers. What should i do? I Love your App, but without Whitelist it isn't Really functional...

-----Original Message-----

**Comment by raiym** [comment]
I have just checked. It works for me.

Could you please check that you have added `cdn1.smartadserver.com` exactly this way. 

without "/" and "http://"

**Comment by Aushilfsarnie** [comment]
Yes, that is the entry. But Everything of my Whitelist is blocked.

-----Original Message-----

**Comment by Aushilfsarnie** [comment]
Whitelist is in german ad-url-ausschlussliste?!?

-----Original Message-----

**Comment by Aushilfsarnie** [comment]
I have the Problem on m.bild.de die example on every browser (Samsung internet, Boat Browser etc). Cdn1.smartadserver.com is always blocked

-----Original Message-----
